database/schema.sql

```diff
@@ -0,0 +1,31 @@
+DROP DATABASE IF EXISTS impulse;
+
+CREATE DATABASE impulse;
+
+USE impulse;
+
+CREATE TABLE Users (
+  id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  username varchar(40) UNIQUE NOT NULL,
+  password varchar(64) NOT NULL,
+  salt varchar(64) NOT NULL
+);
+CREATE TABLE Sessions (
+  id int NOT NULL AUTO_INCREMENT,
+  cookie_id varchar(64),
```

Comment:
Why does this have an underscore but userId and photoId do not? Let's just stick to one style here for Id's.